# Octocoral-Transcriptomes

This repository contains the transcriptome assemblies generated as part of the following study:

<b> New non-bilaterian transcriptomes provide novel insights into the evolution of coral skeletomes.
Conci Nicola, Wörheide Gert, Vargas Sergio </b>  


\
<b>If you use the data please cite:</b>\
Conci Nicola, Wörheide Gert, Vargas Sergio. 2019. New non-bilaterian transcriptomes provide novel insights into the evolution of coral skeletomes. bioRxiv 594242. https://doi.org/10.1101/594242

\
<b>Assemblies details:</b>

<b>Library Preparation</b>: SENSE mRNA-Seq Library Prep Kit V2 for Illumina (Lexogen)\
<b>Sequencing</b>: Strand Specific 50 bp PE (Illumina HiSeq2000, Illumina NextSeq500)\
<b>Quality control and filtering</b>: FastQC (www.bioinformatics.babraham.ac.uk) and Filter Illumina (Agalma-Biolite)[1]\
<b>Assembly</b>: Trinity v.2.5.1 [2]\
<b>Host-symbiont contigs separation</b>: psytrans.py (https://github.com/sylvainforet/psytrans) \
<b>Coding sequences prediction </b>: TransDecoder v.3.0.1 [3]

\
[1] Dunn CW, Howinson M, Zapata F. (2013) Agalma: an automated phylogenomics workflow. <i>BMC Bioinformatics.</i> 14:330\
[2] Grabherr MG et al (2011) Full-length transcriptome assembly from RNA-Seq data without a reference genome. <i> Nat. Biotechnol.</i> 29:644-652\
[3] Haas BJ et al (2013) De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis. <i> Nat. Protoc.</i> 8:1494-1512
